diff -Naurp dnf-4.10.0-orig/dnf/const.py.in dnf-4.10.0/dnf/const.py.in
--- dnf-4.10.0-orig/dnf/const.py.in     2021-10-20 23:31:48.000000000 -0700
+++ dnf-4.10.0/dnf/const.py.in  2021-12-30 13:01:16.224770937 -0700
@@ -54,4 +54,4 @@ VERSION='@DNF_VERSION@'
 USER_AGENT = "dnf/%s" % VERSION

 BUGTRACKER_COMPONENT=NAME.lower()
-BUGTRACKER='https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=%s' % BUGTRACKER_COMPONENT
+BUGTRACKER='https://bugs.openela.org/'
diff -Naurp dnf-4.10.0-orig/doc/conf.py.in dnf-4.10.0/doc/conf.py.in
--- dnf-4.10.0-orig/doc/conf.py.in      2021-10-20 23:31:48.000000000 -0700
+++ dnf-4.10.0/doc/conf.py.in   2021-12-30 13:01:32.347962335 -0700
@@ -267,5 +267,5 @@ rst_prolog = """
 .. _DNF: https://github.com/rpm-software-management/dnf/
 .. _hawkey: http://rpm-software-management.github.io/hawkey/
 .. _YUM: http://yum.baseurl.org/
-.. _bugzilla: https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=dnf
+.. _bugzilla: https://bugs.openela.org/
 """
diff -Naurp dnf-4.10.0-orig/tests/test_config.py dnf-4.10.0/tests/test_config.py
--- dnf-4.10.0-orig/tests/test_config.py        2021-10-20 23:31:48.000000000 -0700
+++ dnf-4.10.0/tests/test_config.py     2021-12-30 13:02:06.399366560 -0700
@@ -55,8 +55,7 @@ class ConfTest(tests.support.TestCase):
     def test_bugtracker(self):
         conf = Conf()
         self.assertEqual(conf.bugtracker_url,
-                         "https://bugzilla.redhat.com/enter_bug.cgi" +
-                         "?product=Fedora&component=dnf")
+                         "https://bugs.openela.org/")

     def test_conf_from_file(self):
         conf = Conf()
