From e05c700cf386c7e38762be6542967ca494adfce9 Mon Sep 17 00:00:00 2001
From: Evan Goode <mail@evangoo.de>
Date: Fri, 16 May 2025 20:38:56 +0000
Subject: [PATCH 2/8] Print "persist" or "transient" in history info

---
 dnf/cli/output.py | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/dnf/cli/output.py b/dnf/cli/output.py
index 1e12c9b48..4699f7800 100644
--- a/dnf/cli/output.py
+++ b/dnf/cli/output.py
@@ -1777,6 +1777,14 @@ Transaction Summary
             else:
                 print(_("Command Line   :"), old.cmdline)
 
+        if old.persistence == libdnf.transaction.TransactionPersistence_PERSIST:
+            persistence_str = "Persist"
+        elif old.persistence == libdnf.transaction.TransactionPersistence_TRANSIENT:
+            persistence_str = "Transient"
+        else:
+            persistence_str = "Unknown"
+        print(_("Persistence    :"), persistence_str)
+
         if old.comment is not None:
             if isinstance(old.comment, (list, tuple)):
                 for comment in old.comment:
-- 
2.49.0

